hugo/content/tutorials/customizing_cli.md

```diff
@@ -73,7 +73,7 @@ export const parseAndValidate = async (fileName: string): Promise<void> => {
 };
 ```
 
-A good amount of the contents for our custom action are shared with the `generateAction` function. This isn't surprising, given that we still need to set up our language's services, and we still need to be able to extract the AST.
+Some amount of the contents for our custom action are shared with the `generateAction` function. This isn't surprising, given that for example we still need to set up our language's services.
```

Comment:
I'm on board with dropping the note about extracting the AST. However I would also suggest another slight change to the 2nd sentence, dropping for example and going for a simpler sentence.
```suggestion
Some amount of the contents for our custom action are shared with the `generateAction` function. This isn't surprising given that we still need to set up our language's services.
```